docs/architecture/grpc-for-wcf-developers/create-project.md

```diff
@@ -1,67 +1,67 @@
 ---
 title: Create a new ASP.NET Core gRPC project - gRPC for WCF Developers
-description: Learn how to create a gRPC project using Visual Studio or from the command line.
+description: Learn how to create a gRPC project by using Visual Studio or the command line.
 ms.date: 09/02/2019
 ---
 
 # Create a new ASP.NET Core gRPC project
 
-.NET Core comes with a powerful CLI tool, `dotnet`, which enables you to create and manage projects and solutions from the command line. The tool is closely integrated with Visual Studio, so everything is also available through the familiar GUI interface. This chapter will show both ways to create a new ASP.NET Core gRPC project: first with Visual Studio, then with the .NET Core CLI.
+.NET Core comes with a powerful CLI tool, `dotnet`, which enables you to create and manage projects and solutions from the command line. The tool is closely integrated with Visual Studio, so everything is also available through the familiar graphical user interface. This chapter shows both ways to create a new ASP.NET Core gRPC project.
```

Comment:
@shirhatti I think this paragraph is slighty misleading. It's not the CLI that is integrated with VS, it's the SDK, which contains the CLI. 

My suggestion:

```suggestion
The .NET Core SDK comes with a powerful CLI tool, `dotnet`, which enables you to create and manage projects and solutions from the command line. The SDK is closely integrated with Visual Studio, so everything is also available through the familiar graphical user interface. This chapter shows both ways to create a new ASP.NET Core gRPC project.
```